cli/src/pcluster/config/cluster_config.py

```diff
@@ -1379,6 +1405,13 @@ def _validate_new_storage_multiple_subnets(self, compute_subnet_ids, new_storage
             compute_subnet_ids=compute_subnet_ids,
             new_storage_count=new_storage_count,
         )
+        for storage in self.shared_storage:
+            if isinstance(storage, (SharedFsxLustre, ExistingFsxOpenZfs, ExistingFsxOntap)) and storage.is_unmanaged:
+                self._register_validator(
+                    UnmanagedFsxMultiAzValidator,
+                    queues=queues,
+                    fsx_subnets=storage.file_system_subnets,
```

Comment:
isn't `UnmanagedFsxMultiAzValidator` expecting a list of AZ instead of a list of subnets?